### PR TITLE
Unbound: migrate misc/dot to grid view

### PR DESCRIPTION
**Pull request by mimugmail**
This *should* be it without the migration php script.

### CODE REVIEW

**Comment by fichtner** [comment]
ok cobow. if I can manage to get an inline grid working that would be my preference on top

**Comment by mimugmail** [comment]
Up to you, its from a year old branch when it was Unbound plus :)

File: src/opnsense/service/templates/OPNsense/Unbound/core/dot.conf [hunk 1]
```diff
@@ -1,10 +1,10 @@
-{% if not helpers.empty('OPNsense.unboundplus.miscellaneous.dotservers') %}
+{% if not helpers.empty('OPNsense.unbound.dot.dots.dot') %}
```
Comment (fichtner) [inline src/opnsense/service/templates/OPNsense/Unbound/core/dot.conf:1]:
```suggestion
{% if not helpers.empty('OPNsense.unboundplus.dot.dots.dot') %}
```

File: src/opnsense/service/templates/OPNsense/Unbound/core/dot.conf [hunk 2]
```diff
@@ -1,10 +1,10 @@
-{% if not helpers.empty('OPNsense.unboundplus.miscellaneous.dotservers') %}
+{% if not helpers.empty('OPNsense.unboundplus.dot.dots.dot') %}
 server:
   tls-cert-bundle: /etc/ssl/cert.pem
 forward-zone:
   name: "."
   forward-tls-upstream: yes
-{%   for dot in OPNsense.unboundplus.miscellaneous.dotservers.split(',') %}
-  forward-addr: {{ dot }}
+{%   for dot in helpers.toList('OPNsense.unbound.dot.dots.dot') %}
```
Comment (fichtner) [inline src/opnsense/service/templates/OPNsense/Unbound/core/dot.conf:7]:
```suggestion
{%   for dot in helpers.toList('OPNsense.unboundplus.dot.dots.dot') %}
```

**Pull request closed without merging**